packages/cmf-router/src/route.js

```diff
@@ -33,16 +18,16 @@ function registerFunction(id, func) {
 	if (typeof func !== 'function') {
 		throw new Error('registerFunction wait for a function');
 	}
-	registry.addToRegistry(`${CONST.REGISTRY_HOOK_PREFIX}:${id}`, func);
+	cmf.registry.addToRegistry(`${REGISTRY_HOOK_PREFIX}:${id}`, func);
 }
 
 /**
  * return a function from the router configuration
  * @param  {string} id
- * @param  {object} contextcmf context
+ * @param  {object} context cmf context
  */
 function getFunction(id, context) {
-	return registry.getFromRegistry(`${CONST.REGISTRY_HOOK_PREFIX}:${id}`, context);
+	return cmf.registry.getFromRegistry(`${REGISTRY_HOOK_PREFIX}:${id}`, context);
```

Comment:
I think we should take this opportunity to add documentation as why a cmf-user need to register and resolve function from the route configuration.

Reply:
Yep don't see why we have registered func here...
Normally <Route> only make render of a component and give props to it, nop ?

Reply:
It's to support router hooks (onEnter, onLeave).
That's what is already in place. But you're right, a little bit of documentation can be good.